lib/rmg/RmgArea.cpp

```diff
@@ -0,0 +1,395 @@
+/*
+ * RmgArea.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#include "RmgArea.h"
+#include "CMapGenerator.h"
+
+namespace rmg
+{
+
+void toAbsolute(Tileset & tiles, const int3 & position)
```

Comment:
While you might use it to convert relative shifts to actual tiles this actually shifts/moves tileset towards position. I think in mathematics this operation is called translate. Also would be nice to make this method a member of Tileset as it modifies tiles instance anyway.

Reply:
So toAbsolute / toRelative might improve readability. Probably we can leave them as is. But still making them members of Tileset might be a better option